src/groups/mqb/mqbcfg/mqbcfg.xsd

```diff
@@ -271,6 +276,27 @@
     </sequence>
   </complexType>
 
+    <complexType name='TlsConfig'>
```

Comment:
We already have `Tcp` in this xsd, so I decided to use `Tls` here instead of `TLS`. It's not used in parsing, so we can change these both things to uppercase if we want.